docs/concepts/services/fetch.md

```diff
@@ -0,0 +1,241 @@
+};
+
+#[derive(Deserialize, Debug, Clone)]
+pub struct ISSPosition {
+    latitude: String,
+    longitude: String,
+}
+
+#[derive(Deserialize, Debug, Clone)]
+pub struct ISS {
+    message: String,
+    timestamp: i32,
+    iss_position: ISSPosition,
+}
+
+#[derive(Debug)]
+pub enum Msg {
+    GetLocation,
+    ReceiveResponse(Result<ISS, anyhow::Error>),
+}
+
+#[derive(Debug)]
+pub struct FetchServiceExample {
+    ft: Option<FetchTask>,
+    iss: Option<ISS>,
+    link: ComponentLink<Self>,
+    error: Option<String>,
+}
+/// Some of the code to render the UI is split out into smaller functions here to make the code
+/// cleaner and show some useful design patterns.
+impl FetchServiceExample {
+    fn view_iss_location(&self) -> Html {
+        match self.iss {
+            Some(ref space_station) => {
+                html! {
+                    <>
+                        <p>{ "The ISS is at:" }</p>
+                        <p>{ format!("Latitude: {}", space_station.iss_position.latitude) }</p>
+                        <p>{ format!("Longitude: {}", space_station.iss_position.longitude) }</p>
+                    </>
+                }
+            }
+            None => {
+                html! {
+                     <button onclick=self.link.callback(|_| Msg::GetLocation)>
+                         { "Where is the ISS?" }
+                     </button>
+                }
+            }
+        }
+    }
+    fn view_fetching(&self) -> Html {
+        if self.ft.is_some() {
+            html! { <p>{ "Fetching data..." }</p> }
+        } else {
+            html! { <p></p> }
+        }
+    }
+    fn view_error(&self) -> Html {
+        if let Some(ref error) = self.error {
+            html! { <p>{ error.clone() }</p> }
+        } else {
+            html! {}
+        }
+    }
+}
+impl Component for FetchServiceExample {
+    type Message = Msg;
+    type Properties = ();
+
+    fn create(_props: Self::Properties, link: ComponentLink<Self>) -> Self {
+        Self {
+            ft: None,
+            iss: None,
+            link,
+            error: None,
+        }
+    }
+    fn change(&mut self, _props: Self::Properties) -> bool {
+        false
+    }
+    fn update(&mut self, msg: Self::Message) -> bool {
+        use Msg::*;
+
+        match msg {
+            GetLocation => {
+                // 1. build the request
+                let request = Request::get("http://api.open-notify.org/iss-now.json")
+                    .body(Nothing)
+                    .expect("Could not build request.");
+                // 2. construct a callback
+                let callback =
+                    self.link
+                        .callback(|response: Response<Json<Result<ISS, anyhow::Error>>>| {
+                            // split up the response into data about the request's status and the returned
+                            // data from the request
+                            let (_, Json(data)) = response.into_parts();
```

Comment:
Since we're only using the body we should use `into_body` instead.
```suggestion
                            let Json(data) = response.into_body();
```

Reply:
The comment above this line needs to be updated with this change.